lms/djangoapps/course_api/blocks/views.py

```diff
@@ -37,8 +37,19 @@ class BlocksView(DeveloperErrorViewMixin, ListAPIView):
 
     **Parameters**:
 
-        * username: (string) The name of the user on whose behalf we want to
-          see the data.
+        * all_blocks: (boolean) Provide a value of "true" to return all
+          blocks. Returns all blocks only if the requesting user has course
+          staff permissions. Blocks that are visible only to specific learners
+          (for example, based on group membership or randomized content) are
+          all included. If all_blocks is not specified, you must specify the
```

Comment:
@nasthagiri  FYI Also changed the description here to "Blocks that are visible only to specific learners... are ALL included".